src/components/milvusCookieConsent/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import CookieConsent from "react-cookie-consent";
+import * as styles from "./milvusCookieConsent.module.less";
+import { useLocation } from "@reach/router";
+import { initializeAndTrack } from "gatsby-plugin-gdpr-cookies";
+
+function isBrowser() {
+  return typeof window !== "undefined";
+}
+
+function getValue(key, defaultValue) {
+  return isBrowser() && window.localStorage.getItem(key)
+    ? JSON.parse(window.localStorage.getItem(key))
+    : defaultValue;
+}
+
+function setValue(key, value) {
+  window.localStorage.setItem(key, JSON.stringify(value));
+}
+
+function useStickyState(defaultValue, key) {
+  const [value, setter] = React.useState(() => {
```

Comment:
For useState, normally we use like [value,setValue] is more clear for us to know what value we set.